config/index.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
     // auto open browser or not
     openBrowser: true,
     publicPath: '/',
-    port: 8080,
+    port: 3000,
```

Comment:
Unrelated. Why did you change the port?

Reply:
My browser was having some kind of glitch and I had to change the port to get it to stop. Sorry about that.

Reply:
Please remove before I can merge.